pages/[resourceType]/create.tsx

```diff
@@ -3,21 +3,21 @@ import React, { useState } from "react";
 import ResourceCodeEditor from "../../components/ResourceCodeEditor";
 import { Button, Center, Text } from "@mantine/core";
 import { cyanMainShade } from "../_app";
+import { cleanNotifications, showNotification, NotificationProps } from "@mantine/notifications";
+import Link from "next/link";
+import { Check, X } from "tabler-icons-react";
 /**
  * CreateResourcePage is a page that renders a code editor and submit button for creating resources.
+ * When the submit button is clicked, a POST request is sent to the test server. If the request is
+ * successful, a success notification with the new ID as a Link appears. Otherwise, an error notification appears.
  * @returns React node with a ResourceCodeEditor component and a submit Button
  */
 const CreateResourcePage = () => {
   const router = useRouter();
   const { resourceType } = router.query;
   const [codeEditorContents, setCodeEditorContents] = useState("");
   const [hasLintError, setHasLintError] = useState(true);
-
-  //handles what will happen once the submit button is clicked.
-  //For now, it console.logs the contents of the ResourceCodeEditor
-  function editorSubmitHandler() {
-    console.log(codeEditorContents);
-  }
+  const NEW_ID_IN_HEADER_REGEX = new RegExp(`${resourceType}/.{1,50}`);
```

Comment:
Nitpick, but since this is just a constant, you should probably define this outside of the component's definition. Otherwise, a new variable will get initialized on every re-render which isn't necessary

Reply:
ah yep you're right, nvm!